tests/supervisor_purge_free_sims_test.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+"""Test purging of free users "old" simulations.
+
+:copyright: Copyright (c) 2019 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+import pytest
+from pykern.pkcollections import PKDict
+import os
+import time
+
+SECS = 3
+DAYS = 1
+
+
+def setup_module(module):
+    os.environ.update(
+        SIREPO_JOB_SUPERVISOR_JOB_CACHE_SECS=str(SECS),
+        SIREPO_JOB_SUPERVISOR_TEST_PURGE_FREQUENCY_SECS=str(SECS),
+        SIREPO_JOB_SUPERVISOR_PURGE_FREE_AFTER_DAYS=str(DAYS),
+    )
+
+
+def test_myapp_free_user_sim_purged(auth_fc):
+    from pykern import pkio
+    from pykern import pkunit
+    from pykern.pkdebug import pkdp
+    import sirepo.auth
+
+    def _check_run_dir(should_exist=False):
+        def _filter(elem):
+            return 'heightWeightReport' in str(elem)
+
+        f = pkio.sorted_glob(pkunit.work_dir().join(
+            'db',
+            'user',
+            fc.sr_auth_state().uid,
+            'myapp',
+            '*',
+            '*',
```

Comment:
Use `*heightWeightReport*` and remove filter

Or, better, use `walk_tree(_user_dir, file_re('heightWeightReport'))`

You don't need to be so specific, because this is a test, just user_dir.